addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js

```diff
@@ -44,11 +44,11 @@ wTourUtils.registerWebsitePreviewTour('add_to_cart_snippet_tour', {
         ...wTourUtils.selectElementInWeSelectWidget('action_picker_opt', 'Buy Now'),
         ...wTourUtils.clickOnSave(),
         wTourUtils.clickOnElement('add to cart button', 'iframe .s_add_to_cart_btn'),
-        wTourUtils.assertPathName('/shop/payment', 'button[name=o_payment_submit_button]'),
+        wTourUtils.assertPathName('/@/shop/payment', 'iframe button[name=o_payment_submit_button]'),
 
-        wsTourUtils.goToCart({quantity: 4, backend: false}),
-        wsTourUtils.assertCartContains({productName: 'Acoustic Bloc Screens'}),
-        wsTourUtils.assertCartContains({productName: 'Conference Chair (Steel)'}),
-        wsTourUtils.assertCartContains({productName: 'Conference Chair (Aluminium)'}),
+        wsTourUtils.goToCart({quantity: 4, backend: true}),
+        wsTourUtils.assertCartContains({productName: 'Acoustic Bloc Screens', backend: true}),
+        wsTourUtils.assertCartContains({productName: 'Conference Chair (Steel)', backend: true}),
+        wsTourUtils.assertCartContains({productName: 'Conference Chair (Aluminium)', backend: true}),
```

Comment:
Tracking purpose:
This is because since the change of this PR, `post_link` don't alter the context by default (switching from FE to BE or vice versa), it didn't make sense.
So this "quick add to cart" now.. add to cart as it is expected and doesn't change context on top of it. Links/form whatever should just do what they are supposed to without altering the context (FE/BE). It's up to our internal mechanism to detect URLs that should do something special or `post_link` which should alter the context should be explicitly flagged as such, which is what this PR does with the 404 add page.